### PR TITLE
Add .mailmap file, add programmer humor

### PR DESCRIPTION
**Pull request by josephcagle**

### CODE REVIEW

File: custom_fortunes/computer-programmer_humor [hunk 1]
```diff
@@ -0,0 +1,7 @@
+"There's no place like ~"
+    - Dorothy Gale
+%
+In order to understand recursion you must first understand recursion.
+%
+42
+
```
Comment (laurendc) [inline custom_fortunes/computer-programmer_humor:7]: I'm guessing `42` is meant to be another quote in here? Could we attribute it to the author or the work that it comes from?  I know that we don't have all authors listed for these but in this case, we are talking about the Hitchhiker's Guide to the Galaxy, after all.

**Review by laurendc — CHANGES_REQUESTED** [review]
@heptocat Thoughts on adding a `.mailmap` file? I don't have a strong opinion on it. I could go either way.

**Comment by heptocat** [comment]
@laurendc I honestly don't really know what we would need a .mailmap file for here, but also, I'm inexperienced with these things. Maybe @DarthFloopy can explain why they think it'd be good to have?

File: custom_fortunes/computer-programmer_humor (hunk 1 above)
Comment (heptocat) [reply custom_fortunes/computer-programmer_humor:7]: Works either way I guess. The original FORTUNE has loads of unquoted jokes / sayings, and it's not like a single number is exactly a quote.
Maybe we could quote Deep Thought ;) that'd be funnier.

**Comment by josephcagle** [comment]
I use different email addresses for my GitHub account and my Git email address, so I add lines in a .mailmap file to tell e.g. `git log` to show both addresses as belonging to me (for whenever I do things through github.com instead of Git).
Hopefully that makes sense ... :)

File: custom_fortunes/computer-programmer_humor (hunk 1 above)
Comment (josephcagle) [reply custom_fortunes/computer-programmer_humor:7]: Hmm ... sorry, I don't quite understand
(I will add a Deep Thought quote anyway)

**Comment by heptocat** [comment]
Thanks for clearing that up @DarthFloopy . But I understand that means it does not need to be on our repository. I will delete it and then merge.

File: custom_fortunes/computer-programmer_humor (hunk 1 above)
Comment (josephcagle) [reply custom_fortunes/computer-programmer_humor:7]: I see you merged this PR. Should I make a new one for the `42` stuff or is it not a big deal?
@heptocat

Comment (heptocat) [reply custom_fortunes/computer-programmer_humor:7]: I think it's fine right now. But it's your programmer_humour @DarthFloopy , so of course make changes or additions as you see fit in a new PR

Comment (laurendc) [reply custom_fortunes/computer-programmer_humor:7]: @heptocat Right, but most of those have authors attributed to them when needed. In this case, 42 is a pretty well-known reference to Hitchhiker's Guide. I can see quality of these going really down if we don't at least loosely try to ensure people add sources to quotes when needed, especially when they are short.

Comment (laurendc) [reply custom_fortunes/computer-programmer_humor:7]: I had actually marked this as needing a revision for that reason.

**Pull request merged**